### DISCUSSION TITLE
continous jog problem

### DISCUSSION THREAD

**Discussion by regpye** [arkypita/LaserGRBL#1224] [Q&A]
I set version 3.8.6 to use the continuos jog and found it doesn't work properly.
Both X and Y will only move in a positive direction, no matter which button is pressed. 
If the home is pressed the carriage will go back as normal.
Changed back to the normal setting and the controls work correctly.

**Comment by StuartB4** [comment]
Seems to work ok for me, did you read this bit and set the machines maximum  travel size?

![image](https://user-images.githubusercontent.com/16980026/103135588-3ce92480-46b1-11eb-85bd-a9213bd8ac45.png)
